diff --git a/node_modules/react-native-ffmpeg/android/build.gradle b/node_modules/react-native-ffmpeg/android/build.gradle
index 7802802..6ecbece 100644
--- a/node_modules/react-native-ffmpeg/android/build.gradle
+++ b/node_modules/react-native-ffmpeg/android/build.gradle
@@ -24,11 +24,11 @@ buildscript {
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 33
 
     defaultConfig {
         minSdkVersion safeExtGet('reactNativeFFmpegPackage', 'https').contains("-lts") ? 16 : 24
-        targetSdkVersion 29
+        targetSdkVersion 33
         versionCode 52
         versionName "0.5.2"
         project.archivesBaseName = "react-native-ffmpeg"
